src/message_interface/message.rs

```diff
@@ -126,8 +125,8 @@ pub enum Message {
     StartBackgroundSync {
         /// Sync options
         options: Option<SyncOptions>,
-        /// Interval
-        interval: Option<Duration>,
+        /// Interval in milliseconds
+        interval_ms: Option<u64>,
```

Comment:
Should have the rename?
```suggestion
        #[serde(rename = "intervalMs")]
        interval_ms: Option<u64>,
```
Also in nodejs it's `intervalInMilliseconds`, align!